### PR TITLE
Print a pretty summary when GoFSH completes

### PR DESCRIPTION
**Pull request by cmoesel**
The subject says it all.

### CODE REVIEW

**Comment by cmoesel** [comment]
I just pushed another commit that also tweaks the roundtrip box to better complement the summary box -- so please also try w/ a round trip to ensure it looks decent.

**Review by jafeltra — COMMENTED** [review]
This looks great! I like the blue color! I had one super minor comment below, but otherwise, the boxes looked nice with and without fishing trip.

File: src/utils/fshingTrip.ts [hunk 1]
```diff
@@ -13,11 +13,13 @@ export function fshingTrip(inDir: string, outDir: string, useLocalSUSHI: boolean
   // Make a pretty box to let the user know we are going into SUSHI mode
   // NOTE: If we add a box to the end of GoFSH output, it may make sense to modify this
   // so we don't have double boxes
```
[the remark this answers is not in the document]
Comment (cmoesel) [reply src/utils/fshingTrip.ts:15]: Oh... right...

**Review by ngfreiter — APPROVED** [review]
This looks good to me. The only thing I noticed is that the box for GoFSH is pretty wide. Like wide enough that the default size of my terminal is too small to properly display it, and then the box is no longer a pretty box:
image
 
I'm approving in case you decide you don't want to change this, but maybe we could do two rows of reporting results?

**Comment by jafeltra** [comment]
I'm not sure if this is the same for Nick, but it looks like the default width of my terminal is 80 characters, so the box doesn't fit for me either. I just hadn't tried that.

**Comment by cmoesel** [comment]
@ngfreiter @jafeltra -- I updated the table in the box to use two rows.  Since we only had 7 items to count, I had to get creative in order to keep the symmetry.

**Review by jafeltra — APPROVED** [review]
That commit is probably in my top 5 favorite commits. I should keep a real list.

I also really like the box width now. It looks nice on my 80 character default size terminal. And I really like that it's the same width as the SUSHI box. I was really working to let that go before. ☺️

**Review by ngfreiter — APPROVED** [review]
Woo! This is awesome!

**Pull request merged**